---
int64 num_of_markers
float64[] x_position
float64[] y_position
float64[] z_position